src/summon/browzine-summon-adapter.js

```diff
@@ -197,14 +246,20 @@ browzine.summon = (function() {
 
       var browzineWebLink = getBrowZineWebLink(data);
       var coverImageUrl = getCoverImageUrl(scope, data, journal);
-      var browzineEnabled = getBrowZineEnabled(scope, data, journal);
+      var defaultCoverImage = isDefaultCoverImage(coverImageUrl);
+      var directToPDFUrl = getDirectToPDFUrl(scope, data);
+
+      if(directToPDFUrl && isArticle(scope) && showDirectToPDFLink()) {
+        var template = directToPDFTemplate(directToPDFUrl);
+        $(documentSummary).find(".docFooter .row:eq(0)").prepend(template);
+      }
 
       if(browzineWebLink) {
-        var template = buildTemplate(scope, browzineWebLink);
+        var template = browzineWebLinkTemplate(scope, browzineWebLink);
         $(documentSummary).find(".docFooter .row:eq(0)").append(template);
       }
 
-      if(coverImageUrl && browzineEnabled) {
+      if(coverImageUrl && !defaultCoverImage) {
         $(documentSummary).find(".coverImage img").attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", "1px 1px 2px #ccc");
```

Comment:
> is this an optimization to reduce the number of times we're trying to set the cover image

`browzineEnabled` stops the default cover image from appearing. But we want to expand Journal Cover coverage to show any Journal Covers we have available. The presence of the default cover image means we do not have the actual Journal Cover, hence the new `defaultCoverImage` check.

Reply:
> But we want to expand Journal Cover coverage to show any Journal Covers we have available.

Right - in the past, we would show our default cover image in Primo/Summon, correct?  

But now, with this change, we will _not_ show our default cover image there, and we will just leave whatever Primo/Summon use by default... if I'm reading this right, anyway.

Am I grokking it? 🤔

Reply:
> in the past, we would show our default cover image in Primo/Summon, correct? 

No. Because the `browzineEnabled` flag seemed to always come with an actual Journal Cover.